tests/integration/test_multiple.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+from typing import List
+
+import pytest
+
+from . import utils
+
+
+@pytest.mark.parametrize(
+    "test_params, check_cmd, expected_cmd_output",
+    [
+        pytest.param(
+            utils.TestParameters(
+                branch="multiple/gomod-and-npm",
+                packages=(
+                    ({"type": "gomod", "path": "gomod-package"}),
+                    {"type": "npm", "path": "npm-package"},
+                    # using RPM to prefetch both package managers
```

Comment:
This feels like naming clash -- Hermeto's package managers are a different entity. WDYT about `# using RPM to provide gomod and npm`?